hack/generate-crd.sh

```diff
@@ -30,7 +30,7 @@ cp manifests/patches/${KUSTOMIZATIONS_FILE} ${MANIFESTS_DIR}
 echo "---------------------------------------------------------------"
 echo "Kustomize: Patch CRDs for singeltons"
 echo "---------------------------------------------------------------"
-oc kustomize "${MANIFESTS_DIR}" | \
+bin/oc kustomize "${MANIFESTS_DIR}" | \
```

Comment:
sorry for the late comment, but this change fixes the `oc` to be in `bin/` folder 
the make target `openshift-client`
```
openshift-client:    
    @type -p oc > /dev/null || bash hack/get-openshift-client.sh 
```
checks the presence  of `oc` anywhere in path.

May be we can add a print statement which suggests user to add oc to path before starting the download, rather than starting download silently in background